day3/slice_time_image.py

```diff
@@ -47,8 +49,32 @@ def slice_time_image(img, slice_times, TR):
         A new copy of the input image with slice-time interpolation applied
     """
     # Get the image data as an array;
+    data = img.get_data()
     # Make a new empty array "interp_data" to hold the interpolated data;
+    interp_data = np.zeros(data.shape)
+    
     # Do the interpolation;
+    time_offset = slice_times/slice_times.shape * TR
+    print "Time Offsets: %s" % time_offset
```

Comment:
Prints very useful for debugging but generally not good for code you'll use a lot, because the output can get pretty verbose.   General tactic is either:

* just remove them when you have got the code debugged and tested or;
* have a VERBOSE flag somewhere in your file, which can be True or False, and only print when the VERBOSE flag is True.